website/src/blogs/ha-vs-dr-and-ha-c2-b2-for-your-db.md

```diff
@@ -35,7 +35,8 @@ I’ll be working off this example of a Kubernetes environment. As you can see,
 
 Now let’s assume you have similar environments on cloud A and data center B or similar. The idea is that we have multiple locations for similar multi-cluster Kubernetes.
 
-*It may be useful for these examples to keep in mind that the fundamental architecture of a Container Attached Storage solution like OpenEBS is to run within the clusters to provide storage to stateful workloads. Essentially, this is the piece that gives each workload its own file or block access (or s3 if you run Minio on top, for example) irrespective of the underlying cloud volumes, existing storage, disks or other hardware such as NVMe. *[*https://www.cncf.io/blog/2018/04/19/container-attached-storage-a-primer/*](https://www.cncf.io/blog/2018/04/19/container-attached-storage-a-primer/)
+*It may be useful for these examples to keep in mind that the fundamental architecture of a Container Attached Storage solution like OpenEBS is to run within the clusters to provide storage to stateful workloads. Essentially, this is the piece that gives each workload its own file or block access (or s3 if you run Minio on top, for example) irrespective of the underlying cloud volumes, existing storage, disks or other hardware such as NVMe. [https://www.cncf.io/blog/2018/04/19/container-attached-storage-a-primer/](https://www.cncf.io/blog/2018/04/19/container-attached-storage-a-primer/)*
```

Comment:
URL is not italic in the earlier version

Reply:
yes it was. single * will add italic, the closing * for the first was not closed properly so it was coming like this 
![image](https://user-images.githubusercontent.com/75615087/125603800-a1191dbb-05d1-49d7-8a82-c57213bf6352.png)